<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>483</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>483</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>483</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Code Editor</string>
   </property>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="Showlinenumbers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show line numbers</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Keyword</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Current Line</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Built-in</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Declare</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Error</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Error2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quotation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Separator</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Signal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Event</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Outlet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Special</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="DefaultSyntaxHighlighting">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Default syntax highlighting</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Showlineseparators">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show line separators</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbBackground">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>434</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="activateHelp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate help entries and tips</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
